src/connector.cpp

```diff
@@ -4935,31 +4800,20 @@ void HostConnector::hostLoadPreset(const uint8_t preset)
                 continue;
             }
 
-            previousPluginStereoOut = blockdata.meta.numOutputs == 2 || dualmono;
-
             // dealing with connections after this point, only valid if preset is the active one
             if (active)
-            {
-                if (++numLoadedPlugins == 1)
-                    hostConnectBlockToChainInput(row, bl);
-                else
-                    hostConnectBlockToBlock(row, last, bl);
+                numLoadedPlugins++;
```

Comment:
on this project we always use ++ as prefix, it is generally good practice for C++